### DISCUSSION TITLE
Clone count

### DISCUSSION THREAD

**Discussion by HandsOfManos** [ScratchAddons/ScratchAddons#6159] [Help]
Hi,
Sorry if I'm not doing this correctly, first time on Github.
I have the extension installed, I see the right click clean up options and I have the search bar. I do not see a clone counter when I run my scratch program, can anyone tell me if there is a setting I'm meant to turn on?

Cheers
Tim

**Comment by CST1229** [comment]
Click the extension's icon on your browser (might be hidden in an "extensions" menu) to open the popup, then click the cog icon at the top right corner of the popup to open the settings page.

In the popup, search for the "Clone counter" addon (there's a search bar).

**Comment by DNin01** [comment]
The counter appears above the stage in the editor when there is at least one clone present. Otherwise, it will be hidden.
